ansible/playbooks/nginx.yml

```diff
@@ -0,0 +1,39 @@
+---
+- hosts: server2
+  tasks:
+  
+  - name: "Create web root directory"
+    file: 
+      path: /home/vagrant/html/
+      state: directory
+      owner: vagrant
+      group: vagrant
+      mode: 0755
+  
+  - name: "Copy welcome page"
+    copy:  
+      src: /vagrant/ansible/playbooks/files/index.html
```

Comment:
you should not put an absolute path here. copy will look inside files automatically, so try it with simply index.html